trace/LogCorrelation.md

```diff
@@ -54,5 +54,5 @@ and format the sampling decision as "true" or "false".
 
 Some logging frameworks allow the insertion of arbitrary key-value pairs into log entries.  When
 a log correlation implementation inserts tracing data by that method, the key names should be
-"opencensusTraceId", "opencensusSpanId", and "opencensusTraceSampled" by default.  The log
-correlation implementation may allow the user to override the tracing data key names.
+"traceId", "spanId", and "traceSampled" by default.  The log correlation implementation may allow
```

Comment:
probably a good name for traceSampled.. I've seen "sampled" but no strong feelings.